pyproject.toml

```diff
@@ -42,7 +42,7 @@ dependencies = [
   # in minor version updates anytime
   "defusedcsv~=2.0",
   "defusedxml~=0.7.1",
-  "django~=4.2",
+  "django>=4.2,<5.0",
```

Comment:
This is the same as `django~=4.2`?

Reply:
I added that <5.0 to prevent the dependabot from picking up the major release update, but then also saw and used that `dependabot ignore major version` command.. 
Think it can go back to what you suggested

Reply:
The way to tell dependabot to ignore update PRs for Django 5.0 is to use `update-types: [ "version-update:semver-major"] # ignore major for django`, as you already did.

For the installation of rdmo the range `django~=4.2` is actually the same as `django>=4.2,<5.0`, so this change is just a different way to say the same thing.